source/base/data_out_base.cc

```diff
@@ -2982,7 +2982,7 @@ namespace DataOutBase
 
       // normalize the gradient
       double gradient_norm =
-        sqrt(pow(gradient[0], 2.0) + pow(gradient[1], 2.0));
+        std::sqrt(std::pow(gradient[0], 2.0) + std::pow(gradient[1], 2.0));
```

Comment:
`pow`, in the global namespace, is `int pow(int)`, so this is definitely a bug.

Reply:
Nah, that's not true :-)
```
NAME
       pow, powf, powl - power functions

SYNOPSIS
       #include <math.h>

       double pow(double x, double y);
       float powf(float x, float y);
       long double powl(long double x, long double y);

       Link with -lm.
```

Reply:
This is going to drive me crazy: I thought that I had found and fixed a bug related to this in the last year but I cannot find the commit now.

Reply:
a-ha, I remember now: it was `abs`, not `pow`. We were already using `fabs` here so the previous version worked.